bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml

```diff
@@ -0,0 +1,3 @@
+bukashk0zzz_filter:
+    # Enable if you need auto filtering form data before constrain(Validation) check
```

Comment:
isn't there a typo here ?

Reply:
Checked twice. I think no. All correct.

Reply:
Shouldn't it be `constraint` instead of `constrain`?

Reply:
My bad. Updated.